spec/apress/moysklad/presenters/assortment_spec.rb

```diff
@@ -12,60 +12,70 @@
-                        name: 'Конина',
-                        description: 'Вкусная, полезная',
+                        article: '290307',
+                        name: 'Платье со сборкой на боку',
+                        description: "Состав: 90% тенсел, 10% лен\nСтрана-производитель: КИТАЙ",
 
-                        minPrice: 20_000.0,
+                        minPrice: {value: 500_000.0},
                         pathName: 'Первый уровень/Второй уровень',
-                        version: 1,
 
                         stock: 0.0,
                         reserve: 0.0,
                         inTransit: 0.0,
                         quantity: 0.0,
 
                         archived: false,
-                        updated: '2017-12-27 16:11:07',
+                        updated: '2020-09-14 11:01:21.392',
 
-                        modificationsCount: 1,
                         weight: 1_000.0,
-                        volume: 500.0,
-                        vat: 0,
-                        effectiveVat: 0,
+                        volume: 1_000.0,
+                        vat: 10,
+                        effectiveVat: 10,
 
                         meta: {type: 'product'},
-                        buyPrice: {value: 10_000.0},
+                        buyPrice: {value: 450_000.0},
 
                         salePrices: [
-                          {value: 0.0, priceType: 'опт'},
-                          {value: 25_000.0, priceType: 'Розничная цена'}
+                          {
+                            priceType: {
+                              externalCode: "cbcf493b-55bc-11d9-848a-00112f43529a",
+                              id: "56fb6e43-d31d-11e9-0a80-01810016c891",
+                              meta: {
+                                href: "https://api.moysklad.ru/api/remap/1.2/context/companysettings/pricetype/56fb6e43-d31d-11e9-0a80-01810016c891",
+                                mediaType: "application/json",
+                                type: "pricetype"
+                              },
+                              name: "Цена продажи"
+                            },
+                            value: 500_000.0
+                          }
                         ],
 
-                        image: {
-                          title: 'bug',
-                          filename: 'bug.jpg',
-
-                          size: 26_316,
-                          updated: '2017-12-27 14:04:34',
-
+                        images: {
                           meta: {
-                            href: 'https://login:password@online.moysklad.ru/api/remap/1.1/download/' \
-                                  'b878dab8-eaf5-11e7-7a31-d0fd001d9a46',
-                            mediaType: 'application/octet-stream'
+                            href: 'https://api.moysklad.ru/api/remap/1.2/entity/product/875d7130-d31d-11e9-0a80-025a002970c1/images',
+                            size: 2
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.